content/en/observability_pipelines/performance.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Performance
+disable_toc: false
+---
+
+## Overview
+
+Sometimes problems can occur even when you try to ensure your Observability Pipelines deployments are sized for the expected load. For example, an application might start generating more logs than usual, or the downstream service you are sending data to starts responding slower than expected. To address these issues:
+- Observability Pipelines propagates backpressure, which is a signal that events cannot be processed as soon as they are received.
+- Observability Pipelines components also have in-memory buffering in case the component receiving the data is busy.
```

Comment:
```suggestion
- Observability Pipelines components also have in-memory buffering in case another component is busy processing incoming data.
```
same with this one, but feel free to take or leave it, just a suggestion :)

Reply:
Thanks! I'm going to edit "another" to "the next component in the pipeline"